janusgraph-cassandra/src/test/java/org/janusgraph/CassandraStorageSetup.java

```diff
@@ -137,7 +137,7 @@ public static String cleanKeyspaceName(String raw) {
         Preconditions.checkNotNull(raw);
         Preconditions.checkArgument(0 < raw.length());
 
-        if (48 < raw.length() || raw.matches("[^a-zA-Z_]")) {
+        if (48 < raw.length() || raw.matches("^.*[^a-zA-Z0-9_].*$")) {
```

Comment:
With the matches method the regex does need to match the full string. The original doesn't and that causes the issue.